modules/ST7735.js

```diff
@@ -51,106 +52,32 @@ function ST7735() {
         this.csPin.write(1);    // Set CS pin high to end transmission
     }
 
+
     // Send data over SPI
     st7735API.writeData = function(data) {
         this.dcPin.write(1);    // Set DC pin low to indicate data
         this.csPin.write(0);
         this.spiBus.transceive(1, data, "write");
         this.csPin.write(1);
+        //delete (data);
```

Comment:
?

Reply:
commented code is commented :)  I'll delete it thanks.